test/e2e/portforward.go

```diff
@@ -77,7 +81,52 @@ func pfPod(expectedClientData, chunks, chunkSize, chunkIntervalMillis string) *a
 	}
 }
 
-func runPortForward(ns, podName string, port int) (*exec.Cmd, int) {
+type portForwardContext struct {
```

Comment:
Unfortunately, context generally means a [context.Context](https://godoc.org/golang.org/x/net/context#Context) object. I'm ok with the use of it here, but I really advise against using the variable name `ctx` below as that is a common name used to refer to a general [context.Context](https://godoc.org/golang.org/x/net/context#Context) object.

Perhaps a better name for this type would encourage a different variable name. But at the very least, please don't use `ctx`.